README.md

```diff
@@ -93,16 +93,20 @@ FAQ
 
 3. What can I run on Origin?
 
-    Origin is designed to run any existing Docker images.  In addition you can define builds that will produce new Docker images from a Dockerfile.  However the real magic of Origin can be seen when using [Source-To-Image](https://github.com/openshift/source-to-image) builds which allow you to simply supply an application source repository which will be combined with an existing Source-To-Image enabled Docker image to produce a new runnable image that runs your application.  We are continuing to grow the ecosystem of Source-To-Image enabled images and documenting them [here](http://docs.openshift.org/latest/using_images/s2i_images/overview.html). Our available images are:
+    Origin is designed to run any existing Docker images.  Additionally, you can define builds that will produce new Docker images using a `Dockerfile`.
+
+    However, the real magic of Origin is [Source-to-Image (S2I)](https://github.com/openshift/source-to-image) builds, which allow developers to simply provide an application source repository containing code to build and execute.  It works by combining an existing S2I-enabled Docker image with application source to produce a new runnable image for your application.
+
+    We are continuing to grow the [ecosystem of Source-to-Image builder images](https://docs.openshift.org/latest/using_images/s2i_images/overview.html) and it's straightforward to [create your own](https://blog.openshift.com/create-s2i-builder-image/).  Our available images are:
 
     * [Ruby](https://github.com/openshift/sti-ruby)
     * [Python](https://github.com/openshift/sti-python)
-    * [NodeJS](https://github.com/openshift/sti-nodejs)
+    * [Node.js](https://github.com/openshift/sti-nodejs)
     * [PHP](https://github.com/openshift/sti-php)
     * [Perl](https://github.com/openshift/sti-perl)
-    * [Wildfly](https://github.com/openshift/wildfly-8-centos)
+    * Other community-supported images, including [WildFly](https://github.com/openshift-s2i/s2i-wildfly)
```

Comment:
For OpenShift Origin (this repo), the distinction is without a difference
for the audience.

On Mon, Jun 20, 2016 at 6:15 PM, Jonathan Yu <notifications@github.com>
wrote:

> In README.md
> <https://github.com/openshift/origin/pull/8007#discussion_r67776945>:
>
> >      * [PHP](https://github.com/openshift/sti-php)
> >      * [Perl](https://github.com/openshift/sti-perl)
> > -    * [Wildfly](https://github.com/openshift/wildfly-8-centos)
> > +    * Other community-supported images, including [WildFly](https://github.com/openshift-s2i/s2i-wildfly)
>
> @danmcp <https://github.com/danmcp> I just wanted to clarify that WildFly
> is a community image, not an "official" one. Is that a worthwhile
> distinction for the readme, or is it better to revert to the way it was
> before? (I'd still update the WildFly name and the URL, though)
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/8007/files/bc4da8fa99b5d01b492ccd85bada33fcb9cd33dd#r67776945>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p87FYh2x1To4CYmiUOk26QJeg8Mkks5qNxDmgaJpZM4HwhAG>
> .
>